src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -84,6 +84,14 @@ public enum JsonCommentHandling : byte
         Skip = (byte)1,
         Allow = (byte)2,
     }
+
```

Comment:
nit: make sure to auto-generate the ref (i.e. remove the extra new lines).
https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/updating-ref-source.md